cmd/m3/tenant-add.go

```diff
@@ -88,23 +88,36 @@ func addTenant(ctx *cli.Context) error {
 		return nil
 	}
 
-	name := ctx.String("admin_name")
-	email := ctx.String("admin_email")
-	if name == "" && ctx.Args().Get(2) != "" {
-		name = ctx.Args().Get(2)
+	userName := ctx.String("admin_name")
+	userEmail := ctx.String("admin_email")
+	if userName == "" && ctx.Args().Get(2) != "" {
+		userName = ctx.Args().Get(2)
 	}
-	if email == "" && ctx.Args().Get(3) != "" {
-		email = ctx.Args().Get(3)
+	if userEmail == "" && ctx.Args().Get(3) != "" {
+		userEmail = ctx.Args().Get(3)
 	}
 
-	if name != "" && email == "" {
-		fmt.Println("User email is needed")
+	if userName == "" || userEmail == "" {
+		fmt.Println("User name and email is needed")
 		return errMissingArguments
 	}
 
-	err := cluster.AddTenantAction(tenantName, tenantShortName, name, email)
```

Comment:
Since this will change the way we create Tenants we will need to update documentation and add for Operator otherwise it will be broken for development. Or should we still allow it from m3?